### DISCUSSION TITLE
npm ignores npmrc in subproject

### DISCUSSION THREAD

**Discussion by morriq** [renovatebot/renovate#17142] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Hello,

We have subproject in monorepo. This subproject has `.npmrc` with content `workspaces=false`.

Since npm@8.5.0 ignores local npmrc and it's described here: https://github.com/npm/cli/issues/4605

Till 8.5.0 renovatebot worked great. It were bumping package-lock.json in subproject when it had to.

But since 8.5.0 renovatebot prints in first line:

`npm WARN ignoring workspace config at /mnt/renovate/gh/Organisation/Repository/apps/subproject/.npmrc `

so it ignores npmrc, and tries to install and create package-lock.json from monorepository instead of subproject.

Solution for that - add `--no-workspaces` when executing npm install.

--

We use pnpm for monorepository but we have specific case where we need package-lock in subproject of this monorepo.

### Relevant debug logs

-

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
this isn't a renovate bug. You probably need to stick to an older npm version.
